nuxeo-drive-client/nxdrive/data/ui5/js/ndrive-conflicts.js

```diff
@@ -54,16 +54,27 @@ ConflictsController.prototype.updateErrors = function($scope) {
 	$scope.ignoreds = ignoreds;
 }
 ConflictsController.prototype.updateConflicts = function($scope, $interval) {
+    var lastContributorPending = false;
 	self = this;
 	$scope.conflicts = angular.fromJson(drive.get_conflicts());
 	for (var i=0; i<$scope.conflicts.length; i++) {
+		lastContributorPending = lastContributorPending || $scope.conflicts[i].need_refresh;
 		if ($scope.conflicts[i].last_error == "DUPLICATING") {
 			setTimeout( function() {
 				self.updateConflicts($scope, $interval);
 				$scope.$apply();
 			}, 1000);
 		}
 	}
+	if (lastContributorPending) {
```

Comment:
the DS sync is very slow to start with:-) we need an API call to resolve userid to full name and I'd rather do it in the background, than when the DS user is waiting for an UI to come up (or update).
Besides, the set of users to be "resolved" is small, e.g. 10 so it will take (up-to) 10 requests initially; afterwards no more  requests because we check if the userid is in the table before requesting it.  

The other issue is that the API to get all users only returns 50 users...weird, and maybe the same 50 all the time?
BTW, isn't it supposed to be a REST API to get all users? at "https://cpodev1-dm.sharpb2bcloud.com/app1/api/v1/users"? (as to get 1 user should be at .../users/336 - plural).
This means that if my user is not in those top 50, and its first and/or last name changes, the only way to update the mapping is to update the db row every time DS makes a call to "https://cpodev1-dm.sharpb2bcloud.com/app1/api/v1/user/id".  
In this case, it makes sense to keep a timestamp of the last update in the Users table and query for "SELECT * from Users WHERE userid=? and last_update < ?" after requesting GetFileSystemItem.